### ISSUE TITLE
About the performance of the model in training

### ISSUE THREAD

**Issue by Iceland-Leo** [dmarnerides/hdr-expandnet#14]
Hi, I use the datasets provided by a CVPR2020 paper "Single-Image HDR Reconstruction by Learning to Reverse the Camera [Pipeline"](https://github.com/alex04072000/SingleHDR) to train ExpandNet, but the loss value never decreases, and the generated image is all black image. I am sure that I haven't changed anything except the training data. Why is that? Is it the datasets that's causing the problem?

**Comment by Iceland-Leo** [comment]
Strangely, I have the same problem with my own HDR dataset. Additionally, I just use OpenCV version 4.4 instead of 3,  is this the reason for this problem?

**Comment by Iceland-Leo** [comment]
Today, I replace your 5 tone mapping methods with another tone mapping method, and I am surprised to find that the training goes well. But in order to make a fair comparison and get the correct results in your paper, I still want to know why using your 5 tone mapping method can't train the model properly. Could you help me find the reasion?

**Comment by dmarnerides** [comment]
Hello there,

I'm glad it works now. I'm not sure what exactly is going wrong, but it sounds like it's with the tone mapping. I haven't ran the training recently with newer versions of the libraries so that might be the problem. 

In any case, maybe try to run it using a single tone mapper each time from the provided OpenCV ones. This way you might find the specific tone mapper that causes the problem.

Best,
Demetris

**Comment by Iceland-Leo** [comment]
OKay, I will try it.

**Comment by Iceland-Leo** [comment]
Experiments show that different version of OpenCV is the key reason. I recommend that you can emphasize this issue in the "README.md" so that others do not encounter the same problem. Thank you very much for your prompt reply.

**Comment by dmarnerides** [comment]
Thanks for investigating this! I will add a note on the readme.
